tests/sqllogictests/suites/base/05_ddl/05_0000_ddl_create_tables

```diff
@@ -142,7 +142,22 @@ statement error 1065
 create table db2.test6(id Int8, created timestamp  DEFAULT today() + a)
 
 statement ok
-create table db2.test6(id Int8, created timestamp)
+create table db2.test6(id Int8, a Int8 DEFAULT 1 + 2, created timestamp DEFAULT now())
```

Comment:
case:  alter table db2.test6 add column c2 timestamp DEFAULT now())

should be covered